cdc/model/errors.go

```diff
@@ -30,3 +30,10 @@ var (
 	ErrCaptureNotExist       = errors.New("capture not exists")
 	ErrUnresolved            = errors.New("unresolved")
 )
+
+// RunningError represents some running error from cdc components, such as processor.
+type RunningError struct {
+	ID        string `json:"id"`
```

Comment:
Could you name it `json:"code"`?

https://github.com/pingcap/tidb/blob/bd28018526397c3ea5e92ee975fea52ec8d6dda7/docs/design/2020-05-08-standardize-error-codes-and-messages.md

Reply:
good catch, make the `RunningError` compatible with TiDB standard